src/hotrod/impl/consistenthash/ConsistentHashFactory.cpp

```diff
@@ -14,9 +13,6 @@ using infinispan::hotrod::protocol::HotRodConstants;
 std::shared_ptr<ConsistentHash> ConsistentHashFactory::newConsistentHash(uint8_t version) {
     std::shared_ptr<ConsistentHash> result;
     switch (version) {
-        case HotRodConstants::CONSISTENT_HASH_V1:
```

Comment:
then is better to add the throw in the default to get the same behaviour against unexpected "version". Agree?